inference-engine/src/gna_plugin/gna_model_serial.hpp

```diff
@@ -74,27 +78,8 @@ struct ModelHeader {
     uint32_t nRotateRows = 0u;
     uint32_t nRotateColumns = 0u;
 
-
-    struct EndPoint {
-        /**
-         * if scale factor is different then pased into infer , network might need to be requantized
-         */
-        float scaleFactor = 0.f;
-        /**
-         * Offset in bytes of pointer descriptor
-         */
-        uint64_t descriptor_offset = 0ull;
-        /**
-         * Endpoint resolution in bytes.
-         */
-        uint32_t element_size = 0u;
-        /**
-         * Number of elements
-         */
-        uint32_t elements_count = 0u;
-    };
-    EndPoint input;
-    EndPoint output;
+    uint32_t nInputs = 0u;
```

Comment:
i thought we agreed that inputs or outputs endpoints would be serialized just like any other layer?

Reply:
Yes. As we agreed they are using same structure as regular layers.
But because deserialization part is different we need to somehow keep track of amount of inputs and outputs.
Probably it can be implemented another way  by creating adtional layer types.